src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -1148,6 +1164,32 @@ private int GetPromptIndexForPoint(ReadOnlyCollection<SnapshotSpan> sourceSpans,
                 }
                 return index;
             }
+            
+            private bool IsInActivePrompt(SnapshotPoint point)
+            {
```

Comment:
I think you could use ```TextView.BufferGraph.MapDownToFirst``` to find the subject buffer containing the point and then just check if it's ```editableBuffer``` (which I would call ```activeBuffer``` for consistency with the method name.